src/checkers/complexity.js

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+import text from "../words_alpha.json"
+
+const formatFeedback = feedback => {
+//	if (!feedback.warning && !feedback.suggestions) {
+//		return <p></p>
+//	}
+	return (
+		<>
+			<>
+				<h4>Suggestions:</h4>
+				<ul>
+					<p>{feedback}</p>
+				</ul>
+			</>
+		</>
```

Comment:
as a heads up (@Garishtech this is relevant for your PR as well) the "suggestions" format is just what I used because that's how zxcvbn formats its' return value, we don't have to use that for other checkers if it doesn't make sense